Topics/Development_Process/Prompt_Engineering.md

```diff
@@ -68,6 +69,25 @@ The provided prompt is not well-structured for the following reasons:
 3. agueness in Instruction: The second prompt is vague in its instructions, missing out on details such as the level of difficulty of the questions or the specific areas of the text to focus on, which could lead to a mismatch in expectations and actual output.
 
 
+## What LLM shoudl you choose?
```

Comment:
Small spelling error here

Reply:
thank you for pointing that out, I have made the change